apps/frontend/src/app/shared/components/profile/profile.component.ts

```diff
@@ -0,0 +1,201 @@
+import { Component } from '@angular/core';
```

Comment:
![image](https://github.com/Venture-Miner/wagademy/assets/111710515/805181eb-6dd8-4871-95e0-592ffe118072)

Here these figma details are missing, we should be able to see how much of the section is completed

Reply:
Luiz asked to remove this question of completed